styles.css

```diff
@@ -116,6 +124,395 @@ li {
   text-decoration: none;
 }
 
+/* start js */
+
+.work-container {
+  border: 1px solid #cbcdd1;
+  display: grid;
+  grid-template-columns: auto;
+  justify-content: center;
+  align-content: center;
+  text-align: left;
+  width: 95%;
+  margin-top: 15vh;
+  margin-left: auto;
+  margin-right: auto;
+  padding: 50px;
+  gap: 12px;
+  background: #fff;
+  border-radius: 16px;
+}
+
+.modal {
+  position: fixed;
+  display: none;
+  z-index: 1;
+  left: 0;
+  top: 0;
+  height: 100%;
+  width: 100%;
+  overflow: auto;
+  align-items: center;
+  background-color: rgba(0, 0, 0, 0.5);
+}
+
+.work-content-modal {
+  border: 1px solid #cbcdd1;
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  text-align: left;
+  width: 95%;
+  margin-left: auto;
+  margin-right: auto;
+  margin-top: 30px;
+  padding: 20px;
+  gap: 12px;
+  background: #fff;
+  border-radius: 16px;
+  -webkit-backdrop-filter: blur(8px);
+  backdrop-filter: blur(8px);
+}
+
+.portfolio-modal-img img {
```

Comment:
- Check the position of the elements, some of them are outside of the screen. To be responsive they should be placed inside the card while you change the size of the window.

![image](https://user-images.githubusercontent.com/93630700/223739330-7d41f847-826e-4d24-bf03-e86f3f683454.png)